concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1814,14 +1969,17 @@ jobs:
             - -c
             - |
               VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-              CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.atc.basic_auth_password concourse-manifest/concourse-manifest.yml)
+              CONCOURSE_ATC_PASSWORD=$("$VAL_FROM_YAML" jobs.concourse.properties.basic_auth_password concourse-manifest/concourse-manifest.yml)
               export CONCOURSE_ATC_PASSWORD
               echo Looking for non running VMs
-              ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
+              echo 2 | ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
```

Comment:
-s2 doesn't work. '2' is not a name of step. I only get these options in the fly:
```
[hijack command options]
      -j, --job=PIPELINE/JOB        Name of a job to hijack
      -c, --check=PIPELINE/CHECK    Name of a resource's checking container to hijack
      -b, --build=                  Build number within the job, or global build ID
      -s, --step=                   Name of step to hijack (e.g. build, unit, resource name)
      -a, --attempt=                Attempt number of step to hijack. Can be specified multiple times for nested retries
```
I wish I did not have to do this but I did not find a way to tell fly that I want to hijack a 'task' type container instead of 'check':
```
1: build #20, step: run-bosh-cli, type: check
2: build #20, step: run-bosh-cli, type: task
```
These 2 have same pipeline name, job name and step name. You can't currently specify any further details in fly.